e2e/api_test.go

```diff
@@ -242,6 +244,92 @@ func TestE2E_StatusEndpoints(t *testing.T) {
 	delete()
 }
 
+func TestE2E_TaskEndpoints_UpdateEnableDisable(t *testing.T) {
+	t.Parallel()
+
+	srv, err := newTestConsulServer(t)
+	require.NoError(t, err, "failed to start consul server")
+	defer srv.Stop()
+
+	tempDir := fmt.Sprintf("%s%s", tempDirPrefix, "disabled_task")
+	delete, err := testutils.MakeTempDir(tempDir)
+	// no defer to delete directory: only delete at end of test if no errors
+	require.NoError(t, err)
```

Comment:
Seeing how you refactored `apiRequest` to check errors there, it would be a neat test utility for this function too!

Reply:
Ooo, good idea! I have on my to-do list to revisit `testutils` and move some functions there (registering consul services, even apiRequest can probably benefit from a move). I will add this to my list! I'm planning to do this after we release Beta.